docs/changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 2025-03-10
+
+- Add Draupnir as moderation tool.
```

Comment:
Good to see that people are updating the changelog.
However, is it worthwhile to add some docs on using Draupnir and link it here?

Reply:
Sure, added.